fdm-calculator/src/balance/nitrogen/supply/fertilizers/index.ts

```diff
@@ -0,0 +1,54 @@
+import type {
+    FertilizerDetail,
+    FieldInput,
+    NitrogenSupplyFertilizers,
+} from "../../types"
+import { calculateNitrogenSupplyByMineralFertilizers } from "./mineral"
+import { calculateNitrogenSupplyByManure } from "./manure"
+import { calculateNitrogenSupplyByCompost } from "./compost"
+
```

Comment:
you might add the group "other" to avoid errors when a product is not categorized as mineral, manure or compost

Reply:
Yes, but the calculation rules did not provide that other fertilizers than those types contribute to the balance. So in that sense it should be ignored. I would propose to change the type manure or using different definitions